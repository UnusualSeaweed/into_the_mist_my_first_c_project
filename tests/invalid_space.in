 
place
place hello
who 
place A02
place AA
place A1A
place A1
place A1
place A112
place  
place A1A
place A5 
place A10 
place  A1
place Z1
place A99
place Z99
