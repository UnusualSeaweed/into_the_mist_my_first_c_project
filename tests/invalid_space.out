Invalid!
Invalid!
Invalid coordinate
Invalid!
Invalid coordinate
Invalid coordinate
Invalid coordinate
Occupied coordinate
Invalid coordinate
Invalid!
Invalid coordinate
Invalid!
Invalid!
Invalid!
Invalid coordinate
Invalid coordinate
Invalid coordinate
